src/common/Page.ts

```diff
@@ -510,15 +510,17 @@ export class Page extends EventEmitter {
           .catch(debugError);
         return;
       }
-      const session = Connection.fromSession(client).session(event.sessionId);
-      const worker = new WebWorker(
-        session,
-        event.targetInfo.url,
-        this._addConsoleMessage.bind(this),
-        this._handleException.bind(this)
-      );
-      this._workers.set(event.sessionId, worker);
-      this.emit(PageEmittedEvents.WorkerCreated, worker);
+      if (event.targetInfo.type === 'worker') {
```

Comment:
was it a bug?

Reply:
Yeah ^^
Previously, an OOP iframe would have shown up as a worker.